ts/components/WithTooltip.svelte

```diff
@@ -38,17 +38,6 @@
     }
 
     onDestroy(() => tooltipObject?.dispose());
-
-    // hack to update field description tooltips
-    let previousTooltip: string = tooltip;
-    $: if (tooltip !== previousTooltip) {
```

Comment:
Just a reminder: We should test every location, where we use WithTooltip. These components have proven to be very fuzzy, and they broke on me in cases where I thought they shouldn't.

Reply:
They're currently only used in the deck options and tag editor. Both seem to have weathered the change :)